.github/CODEOWNERS

```diff
@@ -57,6 +57,9 @@
 # Reviewers to double check any API changes
 /sdk/**/api/                                    @KrzysztofCwalina @pakrym @tg-msft
 
+# Update AutoRest C#
+/eng/CodeGeneration.targets/                    @pakrym @ShivangiReja
```

Comment:
does it need a slash in the end after .targets?

Reply:
I'm not sure, I added a slash because I saw we added for others.

@chidozieononiwu Do we need a slash at the end?

Reply:
Others are directories and this one is a file, I don't think it's needed.

Reply:
see https://github.com/ShivangiReja/azure-sdk-for-net/blob/Add_CodeOwner/.github/CODEOWNERS#L69

Reply:
Ah, right. Removed!